include/ccf/endpoint_registry.h

```diff
@@ -105,6 +105,15 @@ namespace ccf::endpoints
     return true;
   }
 
+  std::string camel_case(
```

Comment:
I don't want this exported/part of the public API, it doesn't need to be visible outside of endpoint_registry.cpp.

Reply:
It does for tests

Reply:
That's why I kept it in `src/ds/nonstd.h` header and separatly in `src/ds/nonstd.cpp` as impl at the first place, because that is a private header

Reply:
I vote for dropping the unit tests, and moving this to `.cpp` only. It's a single-purpose util function for now, we've tested it works as-expected _currently_, and we don't want to risk any additions to the public API at this stage.

Reply:
The current change is the best of both worlds: still tested but not exported.